src/sql/workbench/services/accountManagement/browser/accountManagementService.ts

```diff
@@ -65,6 +65,8 @@ export class AccountManagementService implements IAccountManagementService {
 		this._removeAccountProviderEmitter = new Emitter<azdata.AccountProviderMetadata>();
 		this._updateAccountListEmitter = new Emitter<UpdateAccountListEventParams>();
 
+		_storageService.onWillSaveState(() => this.shutdown());
```

Comment:
Can `shutdown` be updated to `private` in this file like above or is it still called somewhere else?